roles/kubernetes/node/templates/manifests/kube-proxy.manifest.j2

```diff
@@ -23,6 +23,13 @@ spec:
       requests:
         cpu: {{ kube_proxy_cpu_requests }}
         memory: {{ kube_proxy_memory_requests }}
+    livenessProbe:
+      httpGet:
+        host: 127.0.0.1
+        path: /healthz
+        port: 10256
+      initialDelaySeconds: 15
+      timeoutSeconds: 15
```

Comment:
seems aggressive especially on the initial setup. 
Eventually, can you do a readiness with much more delay and retry?

Reply:
The liveness prope for the api-server is defined likewise:

```failureThreshold: 8
      initialDelaySeconds: 15
      periodSeconds: 10
      successThreshold: 1
      timeoutSeconds: 15```

None of all the sevices like api-server, controller etc. has a readiness?

Reply:
at least we should add 
```
failureThreshold: 8 
```
default is 3

Reply:
That we could agree on. Fixed :)